CHANGELOG.md

```diff
@@ -2,6 +2,12 @@
 
 ### New features since last release
 
+* Added a `QuantumCircuit`/QASM to PennyLane template converter
```

Comment:
"Added the ability to automatically convert Qiskit `QuantumCircuit`s or QASM circuits directly into PennyLane templates. The loaded operations can be used directly inside PennyLane circuits.